src/libs/ReportActionsUtils.js

```diff
@@ -174,6 +175,34 @@ function getLastVisibleMessageText(reportID, actionsToMerge = {}) {
         .substring(0, CONST.REPORT.LAST_MESSAGE_TEXT_MAX_LENGTH);
 }
 
+/**
+ * Filter out any reportActions which should not be displayed
+ * Then, sort by their created timestamp first, and reportActionID second
+ * This is a combination of getSortedReportActions() and filterReportActionsForDisplay()
```

Comment:
@tgolen Any suggestion about this comment? I personally think this is enough to explain what the function is, but probably you have others thoughts 😅

Reply:
Sure, I have a few thoughts!

1. The hack doesn't need to be mentioned (it's mentioned in its own comment below)
2. The part about being a combination of two other methods doesn't seem necessary since it's obvious from looking at the code
3. I think the comment should describe _WHY_ the method needs to be a combination of two other methods (but I have a suggestion below which will remove one of the methods)

I suggest:
> This method returns the report actions that are ready for display in the ReportActionsView. The report actions need to be sorted by created timestamp first, and reportActionID second to ensure they will always be displayed in the same order (in case multiple actions have the same timestamp). This is all handled with getSortedReportActions() which is used by several other methods to keep the code DRY.

Reply:
Thank you for your suggestion!
Duo to what I have mentioned in other replies, we should not remove `filterReportActionsForDisplay` method since it's being used in other places. I will update the comment as you suggested
> 3. I think the comment should describe WHY the method needs to be a combination of two other methods (but I have a suggestion below which will remove one of the methods)